sdcm/cluster_gce.py

```diff
@@ -115,6 +115,10 @@ def get_console_screenshot(self):
         self.log.warning('Method is not implemented for GCENode')
         return ''
 
+    @property
+    def ipv6_ip_address(self):
+        raise NotImplementedError()
```

Comment:
How about adding a note/warning before raising the exception?

``On GCE, VPC networks only support IPv4 unicast traffic. They do not support IPv6 traffic within the network.``

Reply:
Yes, It can be pass as parameter to the exception

Reply:
Done